src/dictionary/manager.ts

```diff
@@ -187,9 +190,18 @@ class SchemaManager {
   loadSchemaByVersion = async (
     name: string,
     version: string,
-  ): Promise<dictionaryEntities.SchemasDictionary> => {
-    const newSchema = await dictionaryRestClient.fetchSchema(this.schemaServiceUrl, name, version);
-    return newSchema;
+  ): Promise<dictionaryEntities.SchemasDictionary | undefined> => {
+    try {
+      const newSchema = await dictionaryRestClient.fetchSchema(
+        this.schemaServiceUrl,
+        name,
+        version,
+      );
+      return newSchema;
+    } catch (err) {
+      L.debug(err.message);
+      return Promise.resolve(undefined);
```

Comment:
you can return undefined without Promise.resolve (no need to use promises with async/await) 

that said, I think it's better to rethrow the error in this case because we don't know why it's failed

Reply:
If we throw error here, migration won't be aborted later which makes the current invalid migration in `open` status, instead we want migration to be `closed`, so still need to return `undefined` here.

Reply:
you can try catch there and abort the migration

Reply:
this is what we want: [
![Screen Shot 2020-07-23 at 3 51 49 PM](https://user-images.githubusercontent.com/8744047/88332399-80072c00-ccfc-11ea-858b-75d06d2111e9.png)
](url)